src/palace.pyx

```diff
@@ -486,11 +513,52 @@ cdef class Context:
         return self.impl.is_supported(to_channel_config(channel_config),
                                       to_sample_type(sample_type))
 
-    # TODO: available resamplers
-    # TODO: default resampler index
+    @property
+    def available_resamplers(self) -> List[str]:
+        """The list of resamplers supported by the context.
+
+        If the `AL_SOFT_source_resampler` extension is unsupported
+        this will be an empty list, otherwise there would be
+        at least one entry.
+        """
+        cdef alure.ArrayView[string] resamplers
+        resamplers = self.impl.get_available_resamplers()
+        return [resampler for resampler in resamplers]
```

Comment:
Also could you please try if something like `list(self.impl.get_available_resamplers())` would work.

Reply:
No, it can't: cannot convert `ArrayView[string]` to Python object.

Reply:
Good to know.